os/arch/arm/src/sidk_s5jt200/Kconfig

```diff
@@ -136,6 +136,8 @@ config SIDK_S5JT200_AUTOMOUNT_USERFS_DEVNAME
 	---help---
 		Specifies the device name (/dev/smart0pX) of the partition
 		for user r/w file system.
+		when SMARTFS_MULTI_ROOT_DIRS config is enabled the device name
+		will be appended by "d1" (/dev/smart0pxd1).
```

Comment:
Is it fixed at d1 itself or can continue with d1, d2 based on the number of MULTI ROOT directories?

Reply:
Presently it is fixed at d1 when SMARTFS_MULTI_ROOT_DIRS is enabled